Invoke-HardeningKitty.ps1

```diff
@@ -1,2289 +1,2262 @@
+            }
+            catch {
+                $ResultText = "Failed to set process mitigation settings"
+                $MessageSeverity = "High"
+            }
+
+            $Message = "Starting Category Microsoft Defender Exploit Guard"
+            Write-Output "`n"                
+            Write-ProtocolEntry -Text $Message -LogLevel "Info"                  
+            
+            $Message = $ResultText
+            Write-ResultEntry -Text $Message -SeverityLevel $MessageSeverity
+        }
+        ElseIf ($ProcessmitigationEnableArray.Count -gt 0 -and $ProcessmitigationDisableArray.Count -eq 0) {
+            $ResultText = "Process mitigation settings set"
+            $MessageSeverity = "Passed"  
+
+            try {
+              $Result = Set-Processmitigation -System -Enable $ProcessmitigationEnableArray 
+            }
+            catch {
+                $ResultText = "Failed to set process mitigation settings"
+                $MessageSeverity = "High"
+            }
+
+            $Message = "Starting Category Microsoft Defender Exploit Guard"
+            Write-Output "`n"                
+            Write-ProtocolEntry -Text $Message -LogLevel "Info"
+           
+            $Message = $ResultText
+            Write-ResultEntry -Text $Message -SeverityLevel $MessageSeverity
+        }
+        ElseIf ($ProcessmitigationEnableArray.Count -eq 0 -and $ProcessmitigationDisableArray.Count -gt 0) {
+            $ResultText = "Process mitigation settings set"
+            $MessageSeverity = "Passed"  
+
+            try {
+              $Result = Set-Processmitigation -System -Disable $ProcessmitigationDisableArray 
+            }
+            catch {
+                $ResultText = "Failed to set process mitigation settings"
+                $MessageSeverity = "High"
+            }
+
+            $Message = "Starting Category Microsoft Defender Exploit Guard"
+            Write-Output "`n"                
+            Write-ProtocolEntry -Text $Message -LogLevel "Info"    
+          
+            $Message = $ResultText
+            Write-ResultEntry -Text $Message -SeverityLevel $MessageSeverity
+        }
+    }
+    
+    Write-Output "`n"
+    Write-ProtocolEntry -Text "HardeningKitty is done" -LogLevel "Info"
+
+    If ($Mode -eq "Audit") {
+
+        # HardeningKitty Score
+        $StatsTotal = $StatsPassed + $StatsLow + $StatsMedium + $StatsHigh
+        $ScoreTotal = $StatsTotal * 4
+        $ScoreAchived = $StatsPassed * 4 + $StatsLow * 2 + $StatsMedium
+        If ($ScoreTotal -ne 0 ) {
+            $HardeningKittyScore = ([int] $ScoreAchived / [int] $ScoreTotal) * 5 + 1
+        }        
+        $HardeningKittyScoreRounded = [math]::round($HardeningKittyScore,2)
+
+        # Overwrite HardeningKitty Score if no finding is passed
+        If ($StatsPassed -eq 0 ) {
+            $HardeningKittyScoreRounded = 1.00
+        }
+
+        If ($StatsError -gt 0) {
```

Comment:
Here we should replace the variable to $Script:StatsError as well, and in the output below